### ISSUE TITLE
minSize bug with 8.0.5

### ISSUE THREAD

**Issue by ozbej** [orefalo/svelte-splitpanes#96]
I have this setup with 8.0.5:
```
<Splitpanes class="default-theme" style="height: 400px">
  <Pane minSize={20} size={55}>
    <Splitpanes dblClickSplitter={false} theme="modern-theme">
      <Pane minSize={20}><div>Test 1</div></Pane>
      <Pane minSize={20}><div>Test 2</div></Pane>
      <Pane minSize={20}><div>Test 3</div></Pane>
    </Splitpanes>
  </Pane>
</Splitpanes>
```

minSize doesn't work correctly with svelte-splitpanes version 8.0.5, it is not correctly blocking the view from not being shrinked down. However, it works in version 0.8.2.

**Comment by orefalo** [comment]
will look into it,

..and I also realized I am working way to late at night... the version should be 0.8.5  damn it

**Comment by orefalo** [comment]
I believe this bug is resolved in version 8.0.6.
Please try

**Comment by orefalo** [comment]
closing , should be fixed
